test/types/objectSearch.test.ts

```diff
@@ -98,10 +98,10 @@ describe('objectSearch', () => {
   it('Properly caches and displays options', async () => {
     const { field, searchTerm, result, props, endpoint } = getComponentDefaults({})
       , tester = (await new Tester(ObjectSearch, { props }).mount() as any)
-      , newEndpoint = `${endpoint}-2`
+      , newEndpoint = `${endpoint}2/`
       ;
 
-    tester.endpoints[`/${newEndpoint}/`] = { results: [result] };
+    tester.endpoints[newEndpoint] = { results: [result] };
```

Comment:
Was this change for style or was there a further technical purpose?

Reply:
Technical. Previously you would pass `lawfirms` and it would call getEndpoint with `/lawfirms/?search=` but if you want to do something like pass a full URL or some other more complicated endpoint, these slashes would be added no matter what. It doesn't help us much on the mighty end, but it makes this a much more limited component.